lms/envs/aws.py

```diff
@@ -589,6 +589,10 @@
 ANALYTICS_DASHBOARD_URL = ENV_TOKENS.get("ANALYTICS_DASHBOARD_URL", ANALYTICS_DASHBOARD_URL)
 ANALYTICS_DASHBOARD_NAME = ENV_TOKENS.get("ANALYTICS_DASHBOARD_NAME", ANALYTICS_DASHBOARD_NAME)
 
+# Analytics API
```

Comment:
Sorry for the back and forth.  These should probably have defaults based on the values in common.py.

Reply:
No worries @fredsmith - let me know if you have any additional thoughts.